packages/squiggle-lang/__tests__/reducer/exports_test.ts

```diff
@@ -0,0 +1,9 @@
+import { testEvalToBe, testEvalToMatch } from "../helpers/reducerHelpers.js";
```

Comment:
very minor, but these kind of seem like parser tests, more than reducer tests?

Reply:
Not exactly, this check happens at "compile" stage (AST -> expression).